Core/HLE/HLE.h

```diff
@@ -36,7 +36,9 @@ enum {
 	// Don't allow the call if dispatch or interrupts are disabled.
 	HLE_NOT_DISPATCH_SUSPENDED = 1 << 9,
 	// Indicates the call should write zeros to the stack (stackBytesToClear in the table.)
-	HLE_CLEAR_STACK_BYTES = 1 << 10
+	HLE_CLEAR_STACK_BYTES = 1 << 10,
+	// Indicates that this call operates in kernel mode.
+	HLE_KERNEL_SYSCALL = 1 < 11,
```

Comment:
Yes, this is the next bit up in the flag.  9, 10, 11.

Basically this is here because sceKernelCreateThread (and many other syscalls) has slightly different behavior when called through ThreadManForKernel, but not really different enough to justify creating an entirely separate function.  Also, we could centrally check that the calling thread is in kernel mode for all applicable syscalls (we're not doing either yet, though.)

-[Unknown]